_posts/2018-08-22-engineering-guiding-principles.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: epic
+title: "Engineering Guiding Principles"
+date: 2018-08-22
+author: [ash]
+categories: [concepts, teams, engineering, culture, meta]
+---
+
+The Artsy Engineering team recently underwent the process of defining our guiding principles; you can read through the pull request [here][pr] and the finished principles [here][principles]. In this blog post, I'd like to use our experience of defining these to answer the following questions:
+
+- Why define engineering guiding principles?
+- What makes principles different from company values?
+- How to define guiding principles?
+
+Let's dive in.
+
+<!-- more -->
+
+## Why Define Engineering Principles?
+
+Artsy's CEO has a [blog post][company_values] that discusses why you should define company values and, unsurprisingly, a lot of the same rationale applies to an engineering team's guiding principles. In summary:
+
+- Undefined principles lead to cultural debt (similar to technical debt, but in terms of a team's culture).
+- Defined values lead to greater empowerment (engineers feel safe making decisions on their own, based on the principles).
+- Defined values reduce the risk of unconscious bias (we help our decisions get made consistently).
+- Values are your brand (less applicable for Artsy Engineering; our [contributions][gh] to the software industry are our brand).
+
+I think each of these reasons applies to defining guiding principles as well. Over the past four years, I've seen the Artsy Engineering team encounter situations related to the reasons outlined above and, while the company values have helped us navigate disagreement, they haven't been entirely satisfactory.
+
+The two biggest shortcomings of the values have been an **inconsistent practice** and a **difficulty making team-based decisions**.
+
+Artsy Engineering supports the business primarily through product teams, which each have their own responsibilities and KPIs. Over time, different teams have developed their own subculture. In earnest, I think this is really cool, but it _has_ eventually led to two teams taking radically different approaches to similar problems, or even repeating work that another team is already working on.
+
+That missing consistency also led to difficult making team-based decisions: how can teams make decisions _as a team_ without having a clear set of guidelines with which to evaluate a decision? As I'll explain in the next section, the Artsy company values often left us without a clear answer.
+
+## What Makes Guiding Principles Different from Company Values?
+
+Artsy's [five core values][values] are as follows:
+
+- Art meets Science
+- People are Paramount
+- Quality Worthy of Art
+- Positive Energy
+- Openness
+
+Those have been an incredibly useful framework for guiding Artsy's growth, for three years now. They're good values. But they are _very_ general, designed specifically to be inclusive of every member of, and every team in, our company.
```

Comment:
three years seems really short for the age of our values - but yeah, started thinking about it offsite 2014, shipped 2-15